### DISCUSSION TITLE
küçük harf, büyük harf işlevlerini yapılandırma üzerine

### DISCUSSION THREAD

**Discussion by volkantash** [turkce-yazilim-konati/yazilimca#32] [Programlama Dilleri]
**küçükharf()**
**harfiküçült()** veya **harfiküçült**
**harfküçült()** veya **()harfküçült**
**harfleriküçült()** veya **()harfleriküçült**

Gibi işevleri (fonksiyonları) kendi aralarında tartıp hangi işlevde olacağını ortaya koymak gerekiyor. Siz ne dersiniz? Hangi işi hangisi ile yerine getirelim?

**Türkçe tabanlı Karamel programlama dilinde tartışılan ilgili konu:** 
https://github.com/erhanbaris/karamel/issues/7

**Comment by Nraltas** [comment]
harfleriküçült() - harfleribüyült() daha iyi olucak gibi. 
Daha sonra ilk harfleri büyültmemiz veya küçültmemiz gerekebilir. Bunun için harf değilde harf-ler kullanılması daha iyi.
